spec/dry/types_spec.rb

```diff
@@ -1,6 +1,14 @@
 # frozen_string_literal: true
 
 RSpec.describe Dry::Types do
+  describe ".loader" do
+    it "can eagerly load this library" do
+      Dry::Types.loader.eager_load
```

Comment:
@flash-gordon this might be nice to add as a "standard" test to each Zeitwerkified library; it also might make sense to shell out to a subprocess that does not load RSpec, since `types.rb` _may_ have successfully loaded in a test context (although Zeitwerk may have complained about `types.rb` not defining a Types class).